sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodsShouldNotHaveIdenticalImplementations.cs

```diff
@@ -53,7 +53,20 @@ protected override bool AreDuplicates(MethodDeclarationSyntax firstMethod, Metho
                 secondMethod.Body != null &&
                 firstMethod.Body.Statements.Count >= 2 &&
                 firstMethod.Identifier.ValueText != secondMethod.Identifier.ValueText &&
+                HaveSameParameters(firstMethod.ParameterList?.Parameters, secondMethod.ParameterList?.Parameters) &&
                 firstMethod.Body.IsEquivalentTo(secondMethod.Body, false);
+
+            bool HaveSameParameters(SeparatedSyntaxList<ParameterSyntax>? leftParameters, SeparatedSyntaxList<ParameterSyntax>? rightParameters)
```

Comment:
C# and VB.NET methods are exactly the same BUT the `ParameterSyntax` object is not coming from the same dll so we cannot share the code.